### PR TITLE
Create signature_verification.md

### PR DESCRIPTION
**Pull request by Karsten12**

### CODE REVIEW

File: signature_verification.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Signature Verification
+
+
+In endpoint-monitor, for each job we care about, we create, update and delete Nginx configurations dynamically.
+At startup time, if there is a signature-secret located at `/var/run/secrets/streams-endpoint-monitor/server-auth/signature-secret`, the function `_has_signature_secret()` in `app.py` will set the boolean value `self._signature` to true in `file_config.py`
```
Comment (ddebrunner) [inline signature_verification.md]: Please describe it as the customer would set a secret, not the implementation details. Ie. they set `signature. secret` in a k8s secret.

Comment (ddebrunner) [reply signature_verification.md]: So maybe a separate page for the external user info??

Comment (Karsten12) [reply signature_verification.md]: Will do

File: docs/signature_auth.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Signature authentication 
+
+
+To limit who can input data into your endpoints, you can optionally create a kubernetes secret such that only POST, PUT or PATCH requests with the correct signature in its HTTP header will be allowed through. If this optional kubernetes secret is not given, or is removed, all requests, regardless of type are automatically allowed by default.
+
+To enable this signature verification, create a kubernetes opaque/generic secret with name `signature-secret` and password being the secret of your choice.
```
Comment (ddebrunner) [inline docs/signature_auth.md]: The secret is called `${NAME}-authentication` and contains a key `signature-secret`

File: docs/signature_auth.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+<img width="394" alt="image" src="https://user-images.githubusercontent.com/3769612/65935654-b6229a80-e3ce-11e9-92ff-a13ace0f7cf6.png">
+
+
+The signature is computed as a SHA1 hmac on the request body, using the secret key, and is stored as a hmac hex digest in the request’s `x-signature` HTTP header.
```
Comment (ddebrunner) [inline docs/signature_auth.md]: Can an example be provided of creating a request, e.g. using Python requests.

**Pull request merged**